gensim/models/normmodel.py

```diff
@@ -68,8 +79,26 @@ def calc_norm(self, corpus):
         self.norms = norms
 
     def normalize(self, bow):
+        """
+        Normalizes a simple count representation.
```

Comment:
PEP 257: docstrings should be commands: "do X", "return Y", instead of "does X", "returns Y".
(here and elsewhere in this PR)